command/plugin_register.go

```diff
@@ -90,6 +91,13 @@ func (c *PluginRegisterCommand) Flags() *FlagSets {
 		Usage:      "Semantic version of the plugin. Optional.",
 	})
 
+	f.StringVar(&StringVar{
```

Comment:
I believe the help needs to be updated for `sha256` above to say that it can reference the container image hash.

Reply:
Good call - I gave most of the help text for this command a refresh after taking a closer look. Especially for `-args` which suggested a comma-separated list which doesn't actually work. Added a test for that too to make sure.

Reply:
I also noticed we didn't have a command line arg for environment variables while I was updating the help text, so I added that too as it's a pretty trivial addition.